deps/README.md

```diff
@@ -61,7 +61,7 @@ cd deps/jemalloc
 Hiredis
 ---
 
-Hiredis is used by Sentinel, `redis-cli` and `redis-benchmark`. Like Redis, uses the SDS string library, but not necessarily the same version. In order to avoid conflicts, this version has all SDS identifiers prefixed by `hi`.
+Hiredis is used by Sentinel, `valkey-cli` and `redis-benchmark`. Like Valkey, uses the SDS string library, but not necessarily the same version. In order to avoid conflicts, this version has all SDS identifiers prefixed by `hi`.
```

Comment:
What about redis-benchmark?

Reply:
it's not fully reverted
![image](https://github.com/valkey-io/valkey/assets/33152820/758f78c5-0276-4b90-b339-c1eb041b7ced)

I guess it's a bug at the moment, will check it out and if true, then will fix it

Reply:
@zuiderkwast I can see that both `valkey-benchmark` and `redis-benchmark` works, tested with `unstable` branch, both didn't provided any errors. Should I leave as it is re `deps/README.md` or should I change to `valkey-benchmark` also?

Reply:
Change to valkey-benchmark please.

I don't know the effect of this code, but I think we need to test `valkey-benchmark -u redis://127.0.0.1:6379` and `valkey-benchmark -u valkey://127.0.0.1:6379`. Both are supposed to work.

Reply:
```suggestion
Hiredis is used by Sentinel, `valkey-cli` and `valkey-benchmark`. Like Valkey, uses the SDS string library, but not necessarily the same version. In order to avoid conflicts, this version has all SDS identifiers prefixed by `hi`.
```